build/DependencyVersions.props

```diff
@@ -17,6 +17,8 @@
     <PlatformAbstractionsVersion>2.0.0-preview2-002093</PlatformAbstractionsVersion>
     <DependencyModelVersion>2.0.0-preview2-002093</DependencyModelVersion>
     <CliCommandLineParserVersion>0.1.0-alpha-142</CliCommandLineParserVersion>
+    <CliProjectJsonMigration>1.1.0-alpha-2127</CliProjectJsonMigration>

```

Comment:
My question is: why do we need two separate properties for these versions? Why not just one?